operator-common/src/main/java/io/strimzi/operator/common/ShutdownHook.java

```diff
@@ -12,14 +12,27 @@
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 
+/**
+ * This shutdown hook ensure that {@code Vertx.close()} is called on a clean shutdown,
+ * which in turn calls the stop method of all running Verticles.
+ *
+ * We add a fixed timeout because Vertx has none for stopping running Verticles.
+ */
 @SuppressFBWarnings("RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE")
 public class ShutdownHook implements Runnable {
-    private static final Logger LOGGER = LogManager.getLogger(ShutdownHook.class.getName());
+    private static final Logger LOGGER = LogManager.getLogger(ShutdownHook.class);
     
     private Vertx vertx;
-    
+    private long timeoutMs;
+
     public ShutdownHook(Vertx vertx) {
         this.vertx = vertx;
+        this.timeoutMs = 120_000;
```

Comment:
Better imho to delegate to the 2-parameter ctor `this(vertx, timeoutMs)`

Reply:
Right, thanks.